pom.xml

```diff
@@ -218,6 +218,12 @@
                 </executions>
             </plugin>
         </plugins>
+        <resources>
```

Comment:
Maybe you should follow other modules about how to get a project version.

Reply:
The side effect of `Class#getPackage().getImplementationVersion()` is that, the runtime implementation will always be null.  Or we should consider it as not a problem?

Reply:
Updated to use implementation version directly, ignore not packaged phase null version value case.